models.py

```diff
@@ -0,0 +1,39 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional, TypedDict
+from dane.provenance import Provenance
+
+
+# returned by callback()
+class CallbackResponse(TypedDict):
+    state: int
+    message: str
+
+
+# These are the types of output this worker (possibly) provides (depending on config)
+class OutputType(Enum):
+    EMOTION_CLASS_PREDICTION = "emotion_class_prediction"
+    MODALITY_CONTRIBUTION = "modality_contribution"
+    PROVENANCE = "provenance"  # produced by provenance.py
+
+
+@dataclass
+class ExplainableEmotionRecognitionInput:
+    state: int  # HTTP status code
+    message: str  # error/success message
+    source_id: str = ""  # <program ID>__<carrier ID>
+    # video_file_path: str  # path to the video file (.mp4 file)
+    # audio_file_path: str  # path to the audio file (.wav file)
+    # audio_tensor_path: str  # path to the audio tensor (.pt file)
+    # text_file_path: str  # text data (.txt file)
+    data_folder_path: str # folder with the video audio and text folders and the tsv files and fake label files. (to be made)
```

Comment:
Make distinction between input and output - files that are created within this worker are not input to the worker